src/openforms/ui/tests/test_header.py

```diff
@@ -0,0 +1,28 @@
+from unittest.mock import patch
+
+from django.urls import reverse
+
+from django_webtest import WebTest
+
+from openforms.config.models import GlobalConfiguration
+from openforms.forms.tests.factories import FormFactory
+
+
+class HeaderTest(WebTest):
+    @patch("openforms.config.models.GlobalConfiguration.get_solo")
+    def test_logo_label(self, get_solo_mock):
+        get_solo_mock.return_value = GlobalConfiguration(organization_name="Orgy")
```

Comment:
you might want to use an organization name that's a bit more innocent :grimacing: